src/Sulu/Bundle/RouteBundle/Command/UpdateRouteCommand.php

```diff
@@ -32,6 +32,7 @@ protected function configure()
     {
         $this->setName('sulu:route:update')
             ->addArgument('entity', InputArgument::REQUIRED)
+            ->addArgument('locale', InputArgument::REQUIRED)
```

Comment:
but it could be that a component restricts the locales to a specific ones (by the manager for example).